packages/icons/src/AgentSettings.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="1176.000000pt" height="1176.000000pt" viewBox="0 0 1176.000000 1176.000000" preserveAspectRatio="xMidYMid meet">
```

Comment:
Can you remove the `width` and `height` attributes and run these files through an optimiser. I like myself https://imageoptim.com/mac

Reply:
Optimized images and removed fields as suggested